test/suites/integration/expiration_test.go

```diff
@@ -109,15 +101,8 @@ var _ = Describe("Expiration", func() {
 		env.Monitor.Reset() // Reset the monitor so that we can expect a single node to be spun up after expiration
 
 		// Eventually the node will be tainted, which means its actively being disrupted
-		Eventually(func(g Gomega) {
-			g.Expect(env.Client.Get(env.Context, client.ObjectKeyFromObject(node), node)).Should(Succeed())
-			_, ok := lo.Find(node.Spec.Taints, func(t corev1.Taint) bool {
-				return t.MatchTaint(&karpv1.DisruptedNoScheduleTaint)
-			})
-			g.Expect(ok).To(BeTrue())
-		}).Should(Succeed())
-
-		env.EventuallyExpectCreatedNodeCount("==", 2)
+		env.EventuallyExpectTaintedNodeCount("==", 1)
+		env.EventuallyExpectCreatedNodeCount("==", 1)
```

Comment:
Shouldn't there still be 2 nodes at the same time periodically? 

Maybe its just not caught within whatever window its repeatedly checking?

Reply:
Not at this time, the node count is driven by monitor, which is reset a couple of lines earlier

Reply:
Oh, yea. Missed that reset.